src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
@lorenzo what do you think of https://github.com/cakephp/cakephp/pull/7491#issuecomment-144737355? CakePHP could use add a QueryInterface that would use https://github.com/php-fig/log as logger. That way any PSR-3 logger implementation would work.